WcaOnRails/app/models/competition.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Yes @jonatanklosko. First I had to add .freeze to a definition and then I had to add the `frozen_string_literal: true`